data/encounter/savage.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Could you add a way to generate encounter data in `exampleSite/static/admin/config.yml`? You can find docs for this [here](https://www.netlifycms.org/docs/intro/)

Reply:
Updated in commit https://github.com/The-Balance-FFXIV/glam/pull/63/commits/6f3ed701a95c5a1704ecec6552bb2c2833a01974